### ISSUE TITLE
RE: Hello and "Updating" to Revert Back to Adafruit_BBIO v1.0.3 for Motor Bridge Cape Support

### ISSUE THREAD

**Issue by silver2row** [beagleboard/bb.org-overlays#72]
Hello,

I was told by this fellow tonight on irc #beagle that this is the place to get things done. Now, although I do not understand everything, I can do some type of switch-a-roo every now and then.

This is what I am saying. I changed some library in Python, Adafruit_BBIO's Adafruit_I2C, to get this Motor Bridge Cape to work. I had some trouble w/ this Cape from Seeed-Studio at the beginning and during the last week while updating Adafruit_BBIO, going to bbb.io/latest-images and getting their latest image, and by whatever else has been transpiring w/out my knowledge w/ updates/upgrades.

So,

In hindsight, I would like to add support for this Cape, the Motor Bridge Cape, to whatever it is that needs to be finalized so that it works "out-of-the-box" w/ our current configurations.

I think I may have changed things to suit specific boards but I may be proved wrong. I would like to supply you and anyone else w/ this info. to do whatever you need to do to make this Cape accessible for people who have purchased it in the past and for people that will purchase it in the future.

Seth

P.S. Please let me know what I need to supply you in order to make my understanding your understanding. Commands and/or changed software is what I can provide and I think I may have changed some extra lines in the uEnv.txt file, too. So, let me know. I will return service.

**Comment by RobertCNelson** [comment]
@silver2row doesn't this cape just use 3 pins?

P9_23 is the reset line, so use "config-pin P9_23 gpio_pu/gpio_pd"..
and BB-UART2 which is:

```
config-pin P9_21 uart
config-pin P9_22 uart
```

Or am i missing something?

**Comment by silver2row** [comment]
Hello...I guess I was wrong. Thank you for your feedback.

Seth

**Comment by silver2row** [comment]
Hello Sir,

Where can I find issues relating to config-pin and how to manage these pins w/ config-pin?

Seth

P.S. I found a couple of sites w/ info. on GitHub.com but I was wondering about my board, the BBG. I do not have uart on those two, specific pins (P9_21 and P9_22). I just ran a cmd, config-pin -l P9.21 to see what was what and then I ran a cmd config-pin -q P9.22 to see what was what. I guess I am in the default state on those two pins.

**Comment by silver2row** [comment]
Hello Again Sir,

You were right about the config-pin stuff. I guess default was uart too. I was unaware of things obviously.

Seth

P.S. Thank you for support, again. I ran config-pin -q P9.22 after setting that pin to uart and it read back uart. Thank you again.

**Comment by silver2row** [comment]
Hello Once More Sir,

Seth here. I just noticed something. Setting up config-pin P9.22 uart allows for uart to be set up as a service on that specific pin but does not keep the uart functionality until I set the service up, right?

Seth

P.S. So, I can use the uart function later in a service so I can keep using that uart pin at P9.22 for whatever the reason (in the future) but I would have to change it manually or stop my service. That is nifty and it works great. Thank you.
